src/bmp.c

```diff
@@ -46,3 +45,54 @@ void write_bmp(char *filename, char rgb[], int length, int width) {
     fclose(fp);
     free(bitmap);
 }
+
+// Define portion of the set to be displayed. The more the decimals, the more inside in.
+const double set_left = -2.5;

```

Comment:
Avoid using globals.
You can create a struct and pass it around when needed as a parameter:
```
struct world {
    double set_left;
    double set_right;
    double set_down;
    double set_up;
};
```
And wherever you call the ismandelbrot function, you create one of those structs with the values of the world and pass it as a parameter.